src/criteria/segments-example.js

```diff
@@ -1,10 +1,10 @@
-/* globals newspackPopupsSegmentsExample */
```

Comment:
WDYT of moving this bit to a `src/prompts` and `src/segments` directory? Should be helpful for debugging

Reply:
How about this? e79cba4